torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -411,6 +411,12 @@ AOTI_TORCH_EXPORT AOTITorchError aoti_torch__scaled_mm_v2(
     int8_t use_fast_accum,
     AtenTensorHandle* ret0);
 
+AOTI_TORCH_EXPORT AOTITorchError aoti_torch_cpu__weight_int8pack_mm(
```

Comment:
Unlike those onednn or mkldnn ops you are working on, for aten ops, we should add them to https://github.com/pytorch/pytorch/blob/main/torchgen/aoti/fallback_ops.py and relying on torchgen to autogen them.

Reply:
Thanks. Updated the code by using torchgen.